src/VaultV2.sol

```diff
@@ -762,18 +782,17 @@ contract VaultV2 is IVaultV2 {
     /// @dev For small losses, the incentive might be small, or even null because of rounding. This is ok as small
     /// losses do not cause issues.
     /// @dev The incentive will be null if the msg.sender isn't allowed to receive shares.
-    /// @dev Returns incentiveShares, loss.
-    function realizeLoss(address adapter, bytes memory data) external returns (uint256, uint256) {
-        require(isAdapter[adapter], ErrorsLib.NotAdapter());
-
+    function realizeLosses() external returns (uint256, uint256) {
```

Comment:
Actually something like "clamp" would be better because it could well be that there was no loss (just too much interest has been distributed), unlike before

Reply:
but this will create a loss on the withdrawable amount for some depositors / on their accounting of how much their shares are worth

Reply:
right, totalAsset loss but not necessarily realAsset loss